.circleci/config.yml

```diff
@@ -6,10 +6,11 @@ default_build_environment: &default_build_environment
         username: codainternaltools
         password: $DOCKERHUB_PASSWORD
 
+setup: true 
```

Comment:
What does this do?

Reply:
Alllows Circle to use two config files since we split them up